src/commands/accounts/selectSubscriptions.ts

```diff
@@ -19,6 +19,17 @@ export async function selectSubscriptions(context: IActionContext): Promise<void
 
             const allSubscriptions = await provider.getSubscriptions(false);
 
```

Comment:
The intended behavior is that if no tenants are selected we would default to showing all subscriptions regardless of the tenant. At first I had it so that no subs would show but that seemed very weird to me. Not sure if you have a different opinion on that. 

I'll add a comment explaining that otherwise.